ui/components/fixture-capability-table.vue

```diff
@@ -52,8 +52,7 @@
         <tr
           v-if="cap.model.helpWanted !== null"
           :key="`cap-${index}-helpWanted`">
-          <td colspan="4" />
-          <td colspan="2">
+          <td colspan="6">
```

Comment:
Is this really necessary? I always quite liked the indentation, even though it limits the space for "help wanted" text. But it makes it clear that the box belongs to only that capability.